_data/sidebars/k8smain-sidebar.yml

```diff
@@ -96,6 +96,11 @@ entries:
       output: web, pdf
       type: page
 
+    - title: Multus
```

Comment:
There is some confusion between this page and the "Using Multiple Networks" section, and someone might not know that Multus has anything to do with multiple networks when trying to browse the docs.  Maybe the "Using Multiple Networks" page could have two sections, one for multiple infrastructure networks and another for multiple in-cluster pod networks, which could then link to the Multus page.

Reply:
Good point. I've renamed the "Using Multiple Networks" page to "Using Multiple Host Networks" and added a clarifying statement to the top of the page. Hopefully that makes it clearer.